pom.xml

```diff
@@ -399,14 +392,39 @@
             <dependency>
                 <groupId>org.junit</groupId>
                 <artifactId>junit-bom</artifactId>
-                <version>5.10.2</version>
+                <version>5.10.3</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
         </dependencies>
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
```

Comment:
How come we didn't need to explicitly add the dependencies for those in the past, but we do now?

Reply:
This is not necessarily a requirement. This allows us to declare dependencies up front such that users can pre-download all dependencies in our pom file. I am ok reverting these changes if we think there is a downside to doing this.

Reply:
Not an issue. Just wondering.